packages/firmware/collection.yaml

```diff
@@ -10,12 +10,12 @@ packages:
       autobump.string_replace: '{ "prefix": "" }'
       autobump.prefix: "prefix"
       autobump.hook: |
-        curl -s -L https://download.opensuse.org/distribution/leap/15.3/repo/oss/$(curl -s -L https://download.opensuse.org/distribution/leap/15.3/repo/oss/repodata/repomd.xml | dasel select -p xml 'repomd.data.[0].location.-href') | gunzip | dasel -r xml -p json | jq '.metadata.package[] | select(.name=="u-boot-rpiarm64") | select(.arch!="src").version | map(.)[2] + "-" +  map(.)[1]' -r | tail -n1
```

Comment:
nope, not for firmwares. we have been already there, and seems Leap ones get more attention, TW got broken for RPIs several times

Reply:
really? How weird, I would not expect that! Reverting this!

Reply:
done!